<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="KomputerekDamian" failures="0" tests="4" name="Default test" time="0.085" errors="0" timestamp="24 wrz 2018 20:08:07 GMT">
  <testcase classname="com.engineering_work.example.TestControl" name="@BeforeTest beforeTest" time="0.026">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.engineering_work.example.TestControl.beforeTest(TestControl.java:63)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.beforeRun(TestRunner.java:529)
at org.testng.TestRunner.run(TestRunner.java:497)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeTest beforeTest -->
  <testcase classname="com.engineering_work.example.TestControl" name="@AfterTest afterTest" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterTest afterTest -->
  <testcase classname="com.engineering_work.example.TestControl" name="TestMobileCurrency" time="0.0">
    <skipped/>
  </testcase> <!-- TestMobileCurrency -->
  <testcase classname="com.engineering_work.example.TestControl" name="TestMobileGold" time="0.0">
    <skipped/>
  </testcase> <!-- TestMobileGold -->
  <testcase classname="com.engineering_work.example.TestControl" name="TestWebCurrency" time="0.0">
    <skipped/>
  </testcase> <!-- TestWebCurrency -->
  <testcase classname="com.engineering_work.example.TestControl" name="TestWebGold" time="0.0">
    <skipped/>
  </testcase> <!-- TestWebGold -->
</testsuite> <!-- Default test -->
